src/swap/utils.ts

```diff
@@ -17,3 +17,11 @@ export function isSwapError(response: unknown): response is SwapError {
     response !== null && typeof response === 'object' && 'error' in response
   );
 }
+
+export function formatTokenAmount(amount: string, decimals: number) {
+  // Convert the string amount to a number using decimals value
+  const numberAmount = Number(amount) / Math.pow(10, decimals);
+  // Round to a maximum of 11 significant digits
+  const roundedAmount = Number(numberAmount.toPrecision(11));
```

Comment:
i round to 11 digits here so that the amount will fit in the input but let me know if there is another standard we should follow

Reply:
Let's merge this, but I would like the team feedback on what's best. In particular from Jordan Frankfurt and Conner.